src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -496,12 +493,12 @@ public function editAction(Request $request, int $productId): Response
     public function deleteAction(int $productId): Response
```

Comment:
Now that the allShops is really explicit and not just a convention, I think it's worth renaming the action here to `deleteFromAllShopsAction` along with its associated route name